.github/ISSUE_TEMPLATE/frontend_report.yml

```
@@ -0,0 +1,75 @@
+name: "🌐 Frontend (Issue / Optimization / Documentation)"
+description: Report an issue specifically related to the website frontend. For other inquiries, please use the Discussions section.
+title: "[Frontend]: "
+labels: "Status: Unconfirmed"
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **IMPORTANT:** Failure to comply with the following guidelines may result in immediate closure.
+        - Prior to submitting, kindly search the closed issues to check if the problem you are reporting has already been addressed and resolved. If you come across a closed issue that pertains to your problem, please leave a comment on that issue instead of creating a new one.
+        - If the problem is related to a bug in the website frontend, kindly check for browser compatibility and ensure the issue occurs in multiple browsers before submitting.
```

```suggestion
        - If the problem is related to a bug in the frontend, kindly check for browser compatibility and ensure the issue occurs in multiple browsers before submitting.
```
I don't think anyone is gonna do this.

---

Yes, I am very aware of that, but hardly nobody will do it, except really technical afine who work very fine.

I based this on what tteck did (first run all scripts in verbose and default settings), probably only 5-20% do that :smile:

---

Still good to have, and be able to point to.
Especially if we can't reproduce the problem.